lib/galaxy/dependencies/update_lint_requirements.sh

```diff
@@ -10,7 +10,7 @@ THIS_DIRECTORY="$(cd "$(dirname "$0")" > /dev/null && pwd)"
 
 update_pinned_reqs() {
     VENV=$(mktemp -d "${TMPDIR:-/tmp}/$1_venv.XXXXXXXXXX")
-    python3.8 -m venv "${VENV}"
+    python3 -m venv "${VENV}"
```

Comment:
That's actually on purpose, so that pip doesn't select packages that don't work on our oldest supported Python version, in this case 3.8, which is also the version we use in the GitHub workflow `.github/workflows/dependencies.yaml` .

Reply:
Oh, OK, then closing it. Thanks.